calc/index.html

```diff
@@ -0,0 +1,188 @@
+<!DOCTYPE html>
+
+<html>
+
+<head>
+
+<meta charset="utf-8">
+<meta name="viewport" content="user-scalable=no, initial-scale=1, maximum-scale=1, minimum-scale=1, width=device-width">
+<title>Greenplum Vmem Calculator</title>
+<link href='images/favicon.ico' rel='shortcut icon' type='image/vnd.microsoft.icon'>
+<link rel="stylesheet" type="text/css" href="style/bootstrap.min.css">
+<link rel="stylesheet" type="text/css" href="style/calc.css">
+
+<link href='http://greenplum.org/images/favicon.ico' rel='shortcut icon' type='image/vnd.microsoft.icon'>
+
+<script src="script/jquery-1.11.1.min.js" type="text/javascript"></script>
+<script type="text/javascript" src="script/bootstrap.min.js"></script>
+
+<link href='https://fonts.googleapis.com/css?family=Roboto:400,300,500,100' rel='stylesheet' type='text/css'>
+
+<script type="text/javascript">
+$(document).ready(function(){
+    $('[data-toggle="tooltip"]').tooltip();   
+});
+
+
+function TriggerEvents() {
+    $( "#GPDB-ParamForm" ).change(function() {
+             CalcVmem();
+    });
+
+}
+
+function CalcVmem() {
+	var GPDBOtherMem = parseInt($('#OtherMemory').val());
+    var RAM = parseInt($('#SystemMemory').val()) * 1024;
+    var SWAP = parseInt($('#SwapMemory').val()) * 1024;
+    var NumOfSegs = parseInt($('#SegsPerNode').val());
+    var USERMEM = (RAM + SWAP - ((7.5 * 1024) + (RAM * 0.05)));
+    var KENRNELMEM = (0.026 * (USERMEM / 1.7));
```

Comment:
Thanks again Daniel,
  I made the changes based on your recommendations.



On Thu, Mar 10, 2016 at 1:47 AM, Daniel Gustafsson <notifications@github.com
> wrote:

> In calc/index.html
> <https://github.com/greenplum-db/greenplum-db.github.io/pull/8#discussion_r55643591>
> :
>
> > +
> > +
> > +function TriggerEvents() {
> > +    $( "#GPDB-ParamForm" ).change(function() {
> > +             CalcVmem();
> > +    });
> > +
> > +}
> > +
> > +function CalcVmem() {
> > +	var GPDBOtherMem = parseInt($('#OtherMemory').val());
> > +    var RAM = parseInt($('#SystemMemory').val()) * 1024;
> > +    var SWAP = parseInt($('#SwapMemory').val()) * 1024;
> > +    var NumOfSegs = parseInt($('#SegsPerNode').val());
> > +    var USERMEM = (RAM + SWAP - ((7.5 * 1024) + (RAM * 0.05)));
> > +    var KENRNELMEM = (0.026 * (USERMEM / 1.7));
>
> Typo in variablename, I think you mean KERNELMEM.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/greenplum-db/greenplum-db.github.io/pull/8/files#r55643591>
> .
>